content/_guides/helpers.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Advice for Helpers
+category: chanop
+---
+
+Communities are healthiest when everyone works together to maintain the space
+and shape the culture. If we leave all the emotional labor for Operators to
+deal with, they will be more likely to
+[burn out](catalyst#remember-to-take-breaks-to-avoid-burn-out).
+
+These are some things non-Operator helpers can keep in mind during challenging
+situations to avoid making more work for others.
+
+## Let small things slide
+
+The best response to minor annoying behavior is often to ignore it. Drawing
+attention to it can encourage more bad behaviour, which is the last thing the
+channel needs.
+
+### Attention seeking
+
+If the bad behaviour is someone wanting attention, responding to the
+attention-seeking proves that it works, and they'll keep doing it. If it
+doesn't work, they'll get bored and probably move on.
+
+### Bad language and frustration
+
+If someone is a bit snappy or using bad words out of frustration, scolding
+them can make them more frustrated.
+
+In a support channel, helping a user with their problem is much more important
+than trying to make them ask in the "right" way. So unless it becomes a habit
+or makes the channel uncomfortable, it's not worth derailing the support
+process.
+
+### Typos and non-fluent language
+
+Don't draw attention to someone else's typos or pick on them for their use of
+abbreviations like 'u', unless they specifically request help with wording.
```

Comment:
Should this be single-quotes, feels like the rest of the doc uses double quotes as the primary quote. Admittedly not entirely sure on English quote rules so might turn out to be the ultimate irony to comment this below this header